corehq/apps/userreports/pillow.py

```diff
@@ -57,9 +57,18 @@ def rebuild_table(self, table):
         table.rebuild_table()
 
     def python_filter(self, doc):
-        # filtering is done manually per indicator set change_transport
+        # filtering is done manually per indicator see change_transport
         return True
 
+    def change_trigger(self, changes_dict):
```

Comment:
actually i see that's not clear from the ticket, but that was what @TylerSheffels told me